project/scripts/bootstrappedOnlyCmdTests

```diff
@@ -109,11 +109,11 @@ echo "testing ./bin/scala with arguments"
 test "$EXPECTED_OUTPUT_ARGS" = "$(cat "$tmp")"
 clear_cli_dotfiles project/scripts
 
-echo "testing i12973"
-clear_out "$OUT"
-./bin/scalac -d "$OUT/out.jar" tests/pos/i12973.scala
-echo "Bug12973().check" | TERM=dumb ./bin/scala -cp "$OUT/out.jar" > "$tmp" 2>&1
-grep -qe "Bug12973 is fixed" "$tmp"
+#echo "testing i12973"
```

Comment:
Was this going to be commented out before merge? If there's a bug/regression let's create an issue an add comment here